calculator.js

```diff
@@ -32,16 +32,29 @@ operationButtons.forEach((button) => {
 
 function calc() {
   b = parseInt(inputFieldEl.innerHTML);
-  switch (operation) {
-    case "sum":
-      inputFieldEl.innerHTML = a + b;
-      break;
-
-    default:
-      break;
+  if (operation === 'sum') {
```

Comment:
I thnk it will be fun to extend logic. What if we hit operation button after second number? example: 2 + 2 + 2